docs/web/getting-started.mdx

```diff
@@ -51,7 +51,7 @@ the types we need on the fly:
 
 ```bash
 $ npm config set @buf:registry https://buf.build/gen/npm/v1
-$ npm install @buf/connectrpc_eliza.connectrpc_es @connectrpc/connect @connectrpc/connect-web
+$ npm install @buf/connectrpc_eliza.bufbuild_es@1.10.0-20230913231627-233fca715f49.1 @connectrpc/connect@"^v1.0.0" @connectrpc/connect-web@"^v1.0.0"
```

Comment:
the package changed here, is that intentional?

Reply:
Nope, good catch.